### ISSUE TITLE
MESOS: typo in configuration

### ISSUE THREAD

**Issue by qiujian16** [kubernetes/kubernetes#18420]
In config.go, there is:
ExplicitReconciliationMaxBackoff   WrappedDuration `gcfg:"explicit-reconciliantion-max-backoff"`
ExplicitReconciliationAbortTimeout WrappedDuration `gcfg:"explicit-reconciliantion-abort-timeout"`

This actually should be
explicit-reconciliation-max-backoff
explicit-reconciliation-abort-timeout

**Comment by k82cn** [comment]
@jdef , please help to close this issue; it was corrected in the latest code.

```
59     ExplicitReconciliationMaxBackoff   WrappedDuration `gcfg:"explicit-reconciliation-max-backoff"`
60     ExplicitReconciliationAbortTimeout WrappedDuration `gcfg:"explicit-reconciliation-abort-timeout"`
```

**Comment by jdef** [comment]
fixed by #18575
